src/main/java/seedu/address/logic/commands/DeleteContactCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+
+/**
+ * Deletes a contact identified using it's displayed index from the address book.
+ */
+public class DeleteContactCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete";
```

Comment:
Command word should be "cdelete", have to handle this commandword in the AddressBookParser also